src/components/ChatEntry.css

```diff
@@ -25,10 +25,6 @@
   margin-bottom: 0.5rem;
 }
 
-.chat-entry .entry-body {
-  line-height: 1.5em;
-}
-
```

Comment:
I removed this because it looked like the bubbles in the App were taller than in the screenshot.